conn/physic/units.go

```diff
@@ -1160,9 +1160,126 @@ type Temperature int64
 
 // String returns the temperature formatted as a string in °Celsius.
 func (t Temperature) String() string {
+	if t < minCelsius || t > maxCelsius {
+		return nanoAsString(int64(t)) + "K"
+	}
 	return nanoAsString(int64(t-ZeroCelsius)) + "°C"
 }
 
+// Set sets the Temperature to the value represented by s. Units are to be
+// provided in "C", "°C", "F" or "K" with an optional SI prefix: "p", "n", "u",
```

Comment:
Farenheit is officially "°F" so it should be supported too.

Reply:
°F supported now.